File: OMSWeb/oms-track-monitor/src/TrackObjects/cluster/types/Cluster.ts
import { Color } from 'src/types/Color'
import { D } from 'src/types/D'

type Cluster = {
  id: number
  color: keyof typeof Color
  logicalId: string
  maxVehicles: number
  segments: number[]
  d: D
}

export { Cluster }

File: OMSWeb/Models/Tracks/Cluster.cs
namespace OMSWeb.Models.Tracks
{
    public class Cluster
    {
        public int Id { get; set; }
        public string LogicalId { get; set; }
        public int MaxVehicles { get; set; }
        public string Color { get; set; }
        public string Segments { get; set; }
    }

    public class CluserStatus : Entities.ClusterStatusEntity
    {
    }
}